node-webservice/react-client/src/Components/Grid.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import GridCell from "./GridCell";
+import "../App.css"
```

Comment:
App.css is already imported in App.jsx and shouldn't need to be imported again here.
It could be good to move the grid styling into it's own css file and then import it here